integrationtest/taskext_cli/main.d

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
+
+    Test for ocean.util.app.ext.TaskExt for CliApp
+
+    Copyright:
+        Copyright (c) 2018 sociomantic labs GmbH.
+        All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE_BOOST.txt for details.
+        Alternatively, this file may be distributed under the terms of the Tango
+        3-Clause BSD License (see LICENSE_BSD.txt for details).
+
+*******************************************************************************/
+
+module integrationtest.taskext_cli.main;
+
+import ocean.transition;
+import ocean.util.app.CliApp;
+import ocean.text.Arguments;
+import ocean.task.Task;
+import ocean.task.Scheduler;
+import ocean.core.Test;
```

Comment:
Minor style: imports can/should be in alphabetical order?

Reply:
Nope, we don't require that. As a rule for automatic formatter that can be an option, but it is not worth bothering for manual effort.